### ISSUE TITLE
Broken pre-push Husky 0.15.0-rc.11

### ISSUE THREAD

**Issue by antoniovj1** [typicode/husky#254]
After using Husky I get the next error:

```
.git/hooks/pre-push: 6: export: 7d246f6046c2897906a0e3f660ff0ac62c7a2369: bad variable name
error: failed to push some refs to 'https://github.com/antoniovj1/xxxx.git'
```
Anyone knows how to solve this?

Thanks!!

**Comment by typicode** [comment]
My bad, I think stdin support requires more testing, please use `rc.9` for the moment (I've changed also `next` tag to point to `rc.9`, until a new version fixes it)

**Comment by typicode** [comment]
I've published a fix `v0.15.0-rc.12` under `husky@next` tag. I hope this time there won't be any problem :sweat_smile:

I've upgraded lowdb to use it and was able to commit and push from the terminal.
https://github.com/typicode/lowdb/blob/master/package.json#L61

Let me know, if you still have issues :)

**Comment by antoniovj1** [comment]
Now works like a charm! Thanks!!
